### ISSUE TITLE
Nested arguments not parsing correctly

### ISSUE THREAD

**Issue by tyvdh** [dupski/json-to-graphql-query#1]
I've got a query with a nested argument but the object keys are getting parsed as strings.

```js
{
  query: {
    allAccounts: {
      __args: {
        or: [
          {accountid: {equalTo: "somestring1"}},
          {accountid: {equalTo: "somestring2"}},
          {accountid: {equalTo: "somestring3"}}
        ]
      },
      nodes: {
        accountid: true,
        balance: true
      }
    }
  }
}
```

```gql
query {
    allAccounts (or: [{"accountid":{"equalTo":"somestring1"}},{"accountid":{"equalTo":"somestring2"}},{"accountid":{"equalTo":"somestring3"}}]) {
        nodes {
            accountid
            balance
        }
    }
}
```

For this to be a valid query `accountid` and `equalTo` would need to not be returned as strings

**Comment by dupski** [comment]
Ah yeah, looks like it's not handling arrays as you'd expect. Will have a look tonight :)

**Comment by dupski** [comment]
Should be fixed in version `1.1.0` just published :)
